### PR TITLE
Add tests for include_if_app_enabled

### PR DESCRIPTION
**Pull request by willbarton**
~~We have the `include_if_app_enabled` function that we use to include satellite app URLs in `cfgov/urls.py` if the app is enabled. Some of our satellite's unit tests rely on URL namespacing to resolve their app's URLs. This change sets the namespace to the app name in the URL `include()` if the app is included.~~

I've ~~also~~ written some tests for `include_if_app_enabled` while I was there.

~~This is part of GHE/CFGOV/platform/issues/798~~

## Changes

- Add tests for include_if_app_enabled

## Checklist

* [x] PR has an informative and human-readable title
* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [x] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

**Comment by willbarton** [comment]
I'm going to hold this for now — namespacing might not be the right approach to the test problem, but it would be good to have the tests merged in regardless.

**Comment by willbarton** [comment]
I've backed out the namespacing change for now, it could have wider implications and I can get around it another way. This PR does still add tests for `include_if_app_enabled`, so it's still useful for that.

File: cfgov/transition_utilities/tests/test_conditional_urls.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        self.assertEqual(mock_include.call_count, 1)
+
+    @override_settings(
+        LEGACY_APP_URLS={'core': False},
+        INSTALLED_APPS=['core'],
+    )
+    @mock.patch('transition_utilities.conditional_urls.wagtail_fail_through')
+    def test_include_if_app_enabled_false_in_legacy_apps(
+            self, mock_wagtail_fail_through):
+        """ Test fail-through if LEGACY_APP_URLS[app] is False """
```
Comment (richaagarwal) [inline cfgov/transition_utilities/tests/test_conditional_urls.py]: Maybe I just don't understand the underlying code very well, but this docstring makes little sense to me :\  What is `fail_through` supposed to be, and if we're testing its result why are we mocking it?

Comment (willbarton) [inline cfgov/transition_utilities/tests/test_conditional_urls.py]: `wagtail_fail_through` just serves a URL via Wagtail. I don't want to test that Wagtail will serve a URL, I just want to test that it reaches the point of asking Wagtail to.

Comment (richaagarwal) [inline cfgov/transition_utilities/tests/test_conditional_urls.py]: Oh got it, could you maybe modify the docstring to say "Test that fail_through gets called if ... " ?

Comment (willbarton) [inline cfgov/transition_utilities/tests/test_conditional_urls.py]: Absolutely 😀

**Pull request merged**